### ISSUE TITLE
BBcode for image description

### ISSUE THREAD

**Issue by git-marijus** [friendica/friendica#8347]
According to https://www.bbcode.org/examples/?id=10 the way to add image description or alt text to images should be:
`[img alt="Lubeck city gate" title="This is one of the medieval city gates of Lubeck"]https://www.bbcode.org/images/lubeck_small.jpg[/img]`

Friendica mostly sends this markup though:
`[img=https://www.bbcode.org/images/lubeck_small.jpg]This is one of the medieval city gates of Lubeck[/img]`

Allthough i agree that when editing text by hand the latter markup is easier to use, i think it would make everyones life easier if the standards would be used where possible.

**Comment by MrPetovan** [comment]
Hi @git-marijus , as the page you linked suggests:
> Remember that bbcode tags are implemented to varying degrees and that with many variations. None of this represents strict standards.

Friendica's BBCodes have been tailored for its specific use, and I don't think we meant to reproduce HTML attributes in BBCode form.

**Comment by annando** [comment]
I guess it should be possible adding that format above as an additional format to the parser for incoming content. But we wouldn't use that format for a longer period actively, to make sure that most active systems would support it.

**Comment by git-marijus** [comment]
I am aware that BBCode can hardly be called a standard and yes, it is possible to adapt the parser to deal with various variations. Consider this issue an attempt to keep the amount of variations to a minimum...

**Comment by wiwie** [comment]
I'm trying to embed an image combed with a hyperlink pointing to a different address (equivalent to `<a href="x"><img src="y" /></a>`) to my profile using bbcode. Is this possible with friendica's implementation?

**Comment by MrPetovan** [comment]
```
[url=x][img]y[/img][/url]
```

**Comment by wiwie** [comment]
Nice, thanks that worked!

**Comment by marcusxss** [comment]
Can this be closed, then?
